AUTHORS

```diff
@@ -124,6 +125,7 @@ Madhu Challa                            madhu@cilium.io
 MaiReo                                  sawako.saki@gmail.com
 Maksym Lushpenko                        iviakciivi@gmail.com
 Manali Bhutiyani                        manali@covalent.io
+mandarjog                               mandarjog@gmail.com
```

Comment:
I'm not sure how this is extracted, but shouldn't it be:
```suggestion
Mandar U Jog                            mjog@google.com
```
as per https://github.com/cilium/cilium/pull/13923/commits/0423d8fad42a08dda8559fa94fd380fed42f5d05?

Reply:
Ah, we are extracting [the author field instead of the Signed-off-by](https://github.com/cilium/cilium/commit/0423d8fad42a08dda8559fa94fd380fed42f5d05.patch). That doesn't look right, does it?

Reply:
It's extracted from the e-mail used in the commits.